client/my-sites/people/invite-people/index.jsx

```diff
@@ -361,7 +359,8 @@ const InvitePeople = React.createClass( {
 									disabled={ this.state.sendingInvites } />
 								<FormSettingExplanation>
 									{ this.translate(
-										'(Optional) You can enter a custom message of up to 500 characters that will be included in the invitation to the user(s).'
+										'(Optional) You can enter a custom message of up to 500 characters ' +
```

Comment:
Interesting idea. Can you keep alignment as it is for multiline text without side effects?
At least on Mozilla site (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), they start next line from the beginning of the line.